### ISSUE TITLE
Doesn't work on Windows?

### ISSUE THREAD

**Issue by bfabio** [kyoh86/go-spdx#6]
Hi, we're using `go-spdx` over at https://github.com/italia/publiccode-parser-go. It looks like the assets don't get embedded on Windows, for some reason:

```console
C:\Users\fabio>go install github.com/italia/publiccode-parser-go/v2/pcvalidate@latest

C:\Users\fabio>pcvalidate
panic: specified file is not found

goroutine 1 [running]:
github.com/kyoh86/go-spdx/spdx.init.0()
        C:/Users/fabio/go/pkg/mod/github.com/kyoh86/go-spdx@v0.0.5-0.20220421143955-2f42f2d4c410/spdx/licenses.go:26 +0x21a
```

Not a big issue anyway, because Windows support is low priority for us.

**Comment by kyoh86** [comment]
Thank you for your info.
I'll try to fix it, but I don't have an environment to develop on Windows.
So maybe it takes a little longer time.

**Comment by kyoh86** [comment]
@bfabio It may be fixed by #7

**Comment by bfabio** [comment]
@kyoh86 wow, that was fast :rocket: 

Confirming it [fixes the issue](https://github.com/italia/publiccode-parser-go/tree/windows), thanks!
